.stylelintignore

```diff
@@ -5,3 +5,22 @@
 # Unignore css and scss files too.
 !**/*.css
 !**/*.scss
+
+# Ignore minifed CSS
+**/*.min.css
+
+# Ignore CSS copied in from elsewhere
+projects/js-packages/components/components/boost-score-graph/style-uplot.scss
+projects/packages/jetpack-mu-wpcom/src/features/newspack-blocks/synced-newspack-blocks
+projects/plugins/crm/css/lib
+projects/plugins/crm/css/semantic-ui
+projects/plugins/crm/sass/_color_studio.scss
+projects/plugins/crm/sass/_daterangepicker.scss
+projects/plugins/crm/sass/font-awesome.scss
+projects/plugins/crm/sass/semantic-ui
+projects/plugins/jetpack/scss/cleanslate.scss
+
+# Ignore test-related CSS files that need further investigation
+projects/js-packages/critical-css-gen/tests/data/page-a
+projects/js-packages/i18n-check-webpack-plugin/tests/fixtures/complex-test/src
```

Comment:
i18n-check-webpack-plugin I'm responsible for, and I can tell you the contents of the file don't matter. It's only there to verify that the plugin doesn't break for CSS-only webpack bundles. So feel free to change it to make stylelint happy.

Reply:
23764c5b81